vtex-apps/paginated-table
# Feature/tests

#### What is the purpose of this pull request?

Add tests for most hooks and some table functionalities

#### How should this be manually tested?

yarn test :)

also check for running example https://anita--philco.myvtex.com/admin/sent-offers/offers/?elements=25&from=0&to=25
the only non-test file that was changed affects pagination so it's important to check if it still works fine

#### Types of changes
